ol-keycloak/oltheme/src/main/resources/theme/ol/login/login-password.ftl

```diff
@@ -0,0 +1,49 @@
+<#import "template.ftl" as layout>
+<@layout.registrationLayout displayMessage=!messagesPerField.existsError('password'); section>
+    <#if section = "header">
+        ${msg("doLogIn")}
+    <#elseif section = "form">
+        <div id="kc-form">
+            <div id="kc-form-wrapper">
+                <form id="kc-form-login" onsubmit="login.disabled = true; return true;" action="${url.loginAction}"
+                      method="post">
+                    <div class="${properties.kcFormGroupClass!} no-bottom-margin">
+                        <label for="password" class="${properties.kcLabelClass!}">${msg("password")}</label>
+                        <div class="${properties.kcInputGroup!}">
+                            <input tabindex="2" id="password" class="${properties.kcInputClass!}" name="password"
+                                   type="password" autocomplete="on" autofocus
+                                   aria-invalid="<#if messagesPerField.existsError('password')>true</#if>"
```

Comment:
This looks like it's either `aria-invalid=""` or `aria-invalid="true"` depending on validity?  

**Suggestion:** Would it be possible to omit the attribute entirely if the field is not yet marked as invalid? 

I don't know if an empty string is problematic, but it's not one of the [4 valid values](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-invalid#values).

Reply:
This is OOTB code.  If it proves to be an issue during functional testing I will update it.